src/parsers/forecast_parser.jl

```diff
@@ -254,7 +254,7 @@ function make_forecast_array(all_components, parsed_forecasts::Vector{Dict})
 function write_to_json(filename,Forecasts_dict)
     for (type_key,type_fc) in Forecasts_dict
         for (device_key,device_dicts) in type_fc
-            stringdata =JSON2.json(device_dicts, 3)
+            stringdata =JSON2.write(device_dicts, 3)
```

Comment:
I still don't think this is correct.  It should fail with the 3.  That being said, this function and the next can likely be deleted.

Reply:
![image](https://user-images.githubusercontent.com/16385323/60925406-dd9b1880-a260-11e9-8ee7-a2eafcbb80db.png)

Reply:
The function above it (make_forecast_array) can also be deleted.  I thought I deleted it in a previous PR.

Reply:
I will leave you to delete these in #266 if you think the forecasting won't need them

Reply:
Ok.  But you're still breaking working code.  Just delete ", 3" in both cases.